### PR TITLE
add Quarantined and Isolated to ZG scraper

### PR DESCRIPTION
**Pull request by maekke**
The data doesn't seem to be available in the repo, where result-themen-14-03-01.csv is fetched. But it's available from the website directly...

### CODE REVIEW

**Comment by simgraworldwide** [comment]
https://github.com/openZH/covid_19/issues/807

File: scrapers/scrape_zg.py [hunk 1]
```diff
@@ -33,6 +33,17 @@
     data[row['Stand']][row['Typ']] = row['Anzahl']
 days = list(data.keys())
 last_day = data[days[-1]]
+
+csv_url = 'https://www.zg.ch/behoerden/gesundheitsdirektion/statistikfachstelle/daten/themen/result-themen-14-03-06-e1.csv'
+d_csv = sc.download(csv_url, silent=True)
+reader = csv.DictReader(StringIO(d_csv), delimiter=',')
+data2 = collections.defaultdict(dict)
```
Comment (metaodi) [inline scrapers/scrape_zg.py:40]: I'd prefer to only use one source, if the GitHub source is no longer updated, just replace it with the new CSV URL from the website, no need for `data` and `data2`.

File: scrapers/scrape_zg.py [hunk 2]
```diff
@@ -49,3 +60,8 @@
     print('ICU:', data[day]['Hospitalisierte in Intensivpflege'])
     print('Recovered:', data[day]['Genesene'])
     print('Deaths:', data[day]['Todesfälle'])
+    date = sc.date_from_text(day).isoformat()
+    if date in data2 is not None and 'Positiv getestete Personen' in data2[date] is not None:
```
Comment (metaodi) [inline scrapers/scrape_zg.py:64]: Where does this come from? I don't see `Positiv getestete Personen` in the CSV and can't see the direct link between this and `isolated`

File: scrapers/scrape_zg.py [hunk 3]
```diff
@@ -49,3 +60,8 @@
...
+        print('Isolated:', data2[date]['Positiv getestete Personen'])
+    if date in data2 and 'Total Personen' in data2[date] is not None:
```
Comment (metaodi) [inline scrapers/scrape_zg.py:66]: Dito, where does this `Total Personen` come from?

**Comment by metaodi** [comment]
Thanks @maekke for this PR, I added some comments.

File: scrapers/scrape_zg.py (hunk 2 above)
Comment (metaodi) [reply scrapers/scrape_zg.py:64]: I didn't see that there are two different CSVs, nevermind.

**Comment by metaodi** [comment]
I'm working on this now, using your code base.

**Comment by metaodi** [comment]
Closing this in favor of #808

**Pull request closed without merging**